bookkeeper-server/src/main/java/org/apache/bookkeeper/net/BookieSocketAddress.java

```diff
@@ -38,11 +40,26 @@
     // Member fields that make up this class.
     private final String hostname;
     private final int port;
+    private final boolean isHostnameIPAddress;
```

Comment:
With “optional” the Boolean shouldn’t be needed anymore

Reply:
removed boolean variable